### PR TITLE
refactor: remove schema definitions

### PR DESCRIPTION
**Pull request by anfern777**
# Description
To make it easier to make changes to schemas, we are removing the use of schema-definitions. This PR replaces external schema-definition links with the linked schema, and updates the schema paths in the expected validation error paths.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (a code change that neither fixes a bug nor adds a feature)
- [ ] Dependencies update
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] ⚠️ I ensured that the changes to the schema will be compatible with [schema-tools library](https://github.com/s1seven/schema-tools)

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -28,7 +86,120 @@
...
+                      "type": "string"
+                    }
+                  },
+                  "required": ["CompanyName"]
+                }
+              ],
+              "required": ["ZipCode", "City", "Country"],
+              "additionalProperties": true
+            },
+            {
```
Comment (eamon0989) [inline schema.json:144]: Here Company is made up of 3 objects, the base Company, CompanyAddress and CompanyIdentifiers. Instead of directly copying them over like you did here, maybe you could copy CompanyAddress and CompanyIdentifiers into the definitions and use a local reference like we did [here](https://github.com/material-identity/EN10168-schemas/blob/eb3e77f0256fbecf26911e1d4dd1e3714c2bfb72/schema.json#L176) in EN10168?

IMO it will make it easier to maintain in the future, as if for example, we want to make a change to CompanyAddress across all the schemas, it will be easier if it is it's own object in the schema definitions.

**Review by eamon0989 — CHANGES_REQUESTED** [review]
The tests seem to be failing in the CI, did you run the tests locally?

**Comment by anfern777** [comment]
strange. Yes, I did. Tests are passing locally.

**Comment by anfern777** [comment]
Actually the Ci is failing for quite some time now... probably this same issue. My changes have to do with schema only, so it's impossible this could have any impact in the rendering (which is what is failing)

**Comment by eamon0989** [comment]
> Actually the Ci is failing for quite some time now... probably this same issue. My changes have to do with schema only, so it's impossible this could have any impact in the rendering (which is what is failing)

I took a look into it, my findings are here: https://github.com/material-identity/CoA-schemas/issues/74
TLDR: comment out the test for valid cert 5 and write a comment above it with a link to the issue.

File: schema.json
Comment (eamon0989) [inline schema.json:1]: Don't forget `RefSchemaUrl`

**Review by eamon0989 — APPROVED** [review]
